vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.cc

```diff
@@ -310,7 +310,8 @@ std::string GenerateVerifyLink(type::ExternalWalletPtr wallet) {
   return url;
 }
 
-type::ExternalWalletPtr ResetWallet(type::ExternalWalletPtr wallet) {
+type::ExternalWalletPtr ResetWallet(type::ExternalWalletPtr wallet,
+                                    const bool manual) {
```

Comment:
I think I'd prefer that we not propagate the `manual` parameter down to these `ResetWallet` functions. A couple of other options:

* We could just leave these `ResetWallet` functions as-is, and set the status to `NOT_CONNECTED` inside of `DisconnectWallet` after calling `ResetWallet` (since it has the `manual` parameter).
* We could update the signature of these functions to take a `type::WalletStatus` `previous_status` parameter instead of a `wallet` parameter, since that's the only information that it needs. For the manual case, it would be called like `ResetWallet(type::WalletStatus::NOT_CONNECTED)` and for the other cases it would be called like `ResetWallet(wallet->status)`

Reply:
resolved!